flocker/control/_model.py

```diff
@@ -418,18 +416,16 @@ def __invariant__(self):
                     return (False, '%r manifestation is not on node' % (app,))
         return (True, "")
 
-    def __new__(cls, **kwargs):
```

Comment:
`Node` doesn't have a hostname anymore... but needs to accept one for backwards compat with existing tests. I will see how many such tests are left.